docs/_posts/2019-12-04-42.2.9-release.md

```diff
@@ -0,0 +1,123 @@
+---
+title: PostgreSQL JDBC Driver 42.2.9 Released
+date: 2019-12-04 14:44:10 +0000
+categories:
+  - new_release
+version: 42.2.9
+---
+**Notable changes**
```

Comment:
Now that CHANGELOG.md is filled, if you what this to be also filled, delete the post (this file) and delete the lines from `## [42.2.9] (2019-12-04)` up to `## [Unreleased]`, to start "again", if every thing is ok, this file will be automatically generated with the notable changes included.